src/main/webapp/src/components/songInfo.js

```diff
@@ -163,17 +191,8 @@ class SongInfo extends React.Component {
             </div>
             <div class="song-entity-analysis">
               <Typography variant="h4">Entity Analysis</Typography>
-              <List>
-                {songInfo.entityAnalysis.map((wordInfo, index) => (
-                  <ListItem key={index + 1}>
-                    {wordInfo.word +
-                      " - " +
-                      wordInfo.salience +
-                      " - " +
-                      wordInfo.category}
-                  </ListItem>
-                ))}
-              </List>
+              <button onClick={loadChartAPI}>Get Entity Analysis results</button>
+              <div id="chart-container"></div>
```

Comment:
Add `ref={ref => loadChartAPI()}` and remove button that you had, so this function will be called after rendering.

More info [here](https://reactjs.org/docs/refs-and-the-dom.html)